.docker/ci/docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+/bin/bash -c "find /opt/ros/crystal/ -name tf2_eigen | xargs rm -rf && source /opt/ros/crystal/setup.bash \
```

Comment:
why are we removing tf2_eigen from the source install? This needs good documentation

Reply:
The main reason behind this was this: https://github.com/AcutronicRobotics/moveit2/issues/19#issuecomment-469208116